lib/dmv.rb

```diff
@@ -1,6 +1,11 @@
 class Dmv
+  attr_reader :name, :address, :phone, :services, :facilities
 
-  def initialize
+  def initialize(details ={})
+    @name = details[:name]
+    @address = details[:address]
+    @phone = details[:phone]
+    @services = []
```

Comment:
These attributes should exist on a Facility! A DMV only needs one attribute `@facilities`